src/docs/product/relay/logging.mdx

```diff
@@ -0,0 +1,50 @@
+---
+title: Logging and Error Reporting
```

Comment:
_nit_: What if we call this page something like "Monitoring"? We can still have metrics separately, of course, but here we can give general high level instructions. WDYT?

Reply:
Sounds good, added the new "Monitoring" page